### ISSUE TITLE
Screen Reader Legend Adds Visible Blank Space Inside The Form

### ISSUE THREAD

**Issue by black-eye** [fluentform/fluentform#272]
Hello,

Versions 4.3.24 and 4.3.23 (Core and PRO) added extra space in the begining of each form. You can see it in the example here https://www.liveteflprague.com/devel/poptavka-overeneho-prekladu/ and the screenshot here:

![obrazek](https://user-images.githubusercontent.com/5634870/217668839-25f963d0-e38e-4dea-9d6d-d506e0b53b31.png)

The extra space wasn't there before. In the code it's clearly visible, that this extra space is caused by the tag Legend, even though it's inline styled to 0 height.

Could you please look into it? Thanks

**Comment by dhrupo** [comment]
Hi there,
Thanks for pointing it out to us. We will fix this in the next update. 
FYI, the issue is occurring because of legend's display is set to table by page builder CSS.
You need to set the legend display to block then it will work as expected.

**Comment by black-eye** [comment]
Thank you for the quick reply and good point with the display: table!
It comes from the theme (it's called Page Builder Framework). I'm not sure why they choose to do so, or wheather it's common thing to do or not.

**Comment by black-eye** [comment]
Just out of curiosity: is it absolutely necessary to to use the inline styles there (both for the Legend and Fieldset)? 
Yes, I would probably won't ever needed to override these settings, but if I had to do so, I wouldn't have a chance (inline styles with !important). It looks like a big overkill to me....

**Comment by dhrupo** [comment]
Yes, it's important otherwise theme/other plugin's CSS precedence will take place and hamper the style. Your case is an example here. And, fieldset and legend are added here due to ensure better form accessibility. In the real world, you might not need to tweak them. If needed, just open a support ticket on our portal. We will definitely assist you to solve the problem.
